tcms/rpc/tests/test_testcase.py

```diff
@@ -138,6 +209,166 @@ def test_update_priority_issue_1318(self):
         self.assertEqual('Given-When-Then', self.testcase.text)
 
+    def test_update_default_tester_accepts_user_id(self):
+        self.assertIsNone(self.testcase.default_tester)
+
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.pk,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
+
+    def test_update_default_tester_should_fail_with_fake_user_id(self):
+        self.assertIsNone(self.testcase.default_tester)
+
+        with self.assertRaises(Fault):
+            self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+                self.testcase.pk,
+                {
+                    'default_tester': self.fake_user_id,
+                }
+            )
+
+        self.testcase.refresh_from_db()
+
+        self.assertIsNone(self.testcase.default_tester)
+
+    def test_update_default_tester_accepts_username(self):
+        self.assertIsNone(self.testcase.default_tester)
+
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.username,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
+
+    def test_update_default_tester_should_fail_with_fake_username(self):
+        self.assertIsNone(self.testcase.default_tester)
+
+        with self.assertRaises(Fault):
+            self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+                self.testcase.pk,
+                {
+                    'default_tester': self.fake_username,
+                }
+            )
+
+        self.testcase.refresh_from_db()
+
+        self.assertIsNone(self.testcase.default_tester)
+
+    def test_update_default_tester_accepts_email(self):
+        self.assertIsNone(self.testcase.default_tester)
+        self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+            self.testcase.pk,
+            {
+                'default_tester': self.new_author.email,
+            }
+        )
+
+        self.testcase.refresh_from_db()
+
+        self.assertEqual(self.new_author.pk, self.testcase.default_tester.pk)
+
+    def test_update_default_tester_should_fail_with_fake_email(self):
+        self.assertIsNone(self.testcase.default_tester)
+
+        with self.assertRaises(Fault):
+            self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+                self.testcase.pk,
+                {
+                    'default_tester': self.fake_email,
+                }
+            )
+
+        self.testcase.refresh_from_db()
+
+        self.assertIsNone(self.testcase.default_tester)
+
+    def test_update_reviewer_accepts_user_id(self):
```

Comment:
Missing assert for initial value of reviewer (or that it is different than new_author)